programs/uxd/src/test/utils/calculate_amount_less_fees.rs

```diff
@@ -0,0 +1,39 @@
+// Unit tests
+#[cfg(test)]
+mod test_calculate_amount_less_fees {
+    use crate::utils::calculate_amount_less_fees;
+    use anchor_lang::Result;
+
+    #[test]
+    fn test_zero_fees() -> Result<()> {
```

Comment:
Might want to use prop_test here instead of arbitrary numbers

Reply:
done 👌